cmd/plugins/topology-aware/policy/topology-aware-policy.go

```diff
@@ -484,6 +484,8 @@ func (p *policy) Reconfigure(newCfg interface{}) error {
 		return policyError("failed to reconfigure: %v", err)
 	}
 
+	p.registerImplicitAffinities()
```

Comment:
since we are here, can we perhaps handle the error instead of ignoring here?

Reply:
Okay. I added an error check to both call sites, `Setup` and `Reconfigure`.